napari/_qt/layers/qt_image_base_layer.py

```diff
@@ -16,6 +17,9 @@ class QtBaseImageControls(QtLayerControls):
     def __init__(self, layer):
         super().__init__(layer)
 
+        # initialize qt events
+        self.events = EmitterGroup()
```

Comment:
```suggestion
        self.events = EmitterGroup(source=self)
```
so the event object will have a `.source` attribute that tracks what created the event.  Maybe also want to take advantage of `auto_connect` ... but haven't thought my way through that yet.